engines/tika/src/main/java/org/alfresco/transform/tika/transformers/PdfBoxTransformer.java

```diff
@@ -24,25 +24,24 @@
  * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
  * #L%
  */
-package org.alfresco.transformer;
+package org.alfresco.transform.tika.transformers;
 
-import org.springframework.boot.test.context.SpringBootTest;
+import org.apache.tika.extractor.DocumentSelector;
+import org.apache.tika.parser.Parser;
+import org.springframework.stereotype.Component;
 
-/**
- * Tests LibreOfficeController with a server test harness.
- */
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class LibreOfficeHttpRequestTest extends AbstractHttpRequestTest
+@Component
+public class PdfBoxTransformer extends AbstractTikaTransformer
 {
     @Override
-    protected String getTransformerName()
+    protected Parser getParser()
     {
-        return "LibreOffice";
+        return tika.pdfParser;
     }
 
     @Override
-    protected String getSourceExtension()
+    protected DocumentSelector getDocumentSelector()
     {
-        return "doc";
+        return tika.pdfBoxEmbededDocumentSelector;
```

Comment:
`pdfBoxEmbededDocumentSelector` is a static field and should be accessed in a static way.

```suggestion
        return Tika.pdfBoxEmbededDocumentSelector;
```

Reply:
done